src/Config_Command.php

```diff
@@ -260,4 +281,45 @@ private static function _read( $url ) {
 		}
 	}
 
+	/**
+	 * Prints the value of a constant or global defined in the wp-config.php file.
+	 *
+	 * If the constant or global is not defined in the wp-config.php file then an error will be returned.
+	 *
+	 * @param array $assoc_args
+	 * @param bool $get_constant
+	 * @param array $wp_config_constants
+	 * @param array $wp_config_vars
+	 *
+	 * @return string The value of the requested constant or global as defined in the wp-config.php file; if the
+	 *                requested constant or global is not defined then the function will print an error and exit.
+	 */
+	private function return_constant_or_global( $assoc_args, $get_constant, $wp_config_constants, $wp_config_vars ) {
+		if ( $get_constant ) {
+			$key       = $assoc_args['constant'];
+			$type      = 'constant';
+			$look_into = $wp_config_constants;
+		} else {
+			$key       = $assoc_args['global'];
+			$type      = 'global';
+			$look_into = $wp_config_vars;
+		}
+
+		$keys = array_column( $look_into, 'key' );
+
+		if ( false !== $index = array_search( $key, $keys ) ) {
+			return $look_into[ $index ]['value'];
+		}
+
+		$candidate = Utils\get_suggestion( $key, $keys );
+
+		if ( empty( $candidate ) ) {
+			WP_CLI::error( "the {$key} {$type} is not defined in the wp-config.php file." );
```

Comment:
In both error messages, "the" at the beginning of the message should be capitalized.